content/metadata.yaml

```
@@ -29,7 +36,14 @@ authors:
     orcid: 0000-0002-6224-0347
     email: josh.shapiro@ccdatalab.org
     contributions:
-      - TODO
+      - Methodology
+      - Software
+      - Validation
+      - Formal analysis
```

I think provision of computing resources applies here as well:
```suggestion
      - Formal analysis
      - Resources
```